gg.minion.Minion.yml

```diff
@@ -0,0 +1,75 @@
+app-id: gg.minion.Minion
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+command: minion.sh
+sdk: org.freedesktop.Sdk
+tags:
+  - proprietary
+finish-args:
+  - --env=PATH=/app/java/jre/bin:/app/bin
+  - --socket=x11
+  - --share=network
+  - --filesystem=home
+  - --filesystem=~/.var/app/com.valvesoftware.Steam
+  - --filesystem=~/.var/app/net.lutris.Lutris
+modules:
+  # Liberica JDK for running Minion as it supports OpenJFX out of the box
+  - name: java-11-runtime
+    buildsystem: simple
+    build-commands:
+      - mkdir -p             /app/java/jre
+      - cp -r {bin,lib,conf} /app/java/jre
+    sources:
+      - type: archive
+        url: https://download.bell-sw.com/java/11.0.17+7/bellsoft-jre11.0.17+7-linux-amd64-full.tar.gz
+        sha256: f3578ddb8fc53053d6c12507c7235e777d6a104d85f8b104fa043c81978d0cb9
+        only-arches:
+          - x86_64
+      - type: archive
+        url: https://download.bell-sw.com/java/11.0.17+7/bellsoft-jre11.0.17+7-linux-aarch64-full.tar.gz
+        sha256: 2a2269e04716ddfb29404a3c5ba8a766b744dbac16c8df9c879adf3ea308d942
+        only-arches:
+          - aarch64
+  - name: minion
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 apply-extra                ${FLATPAK_DEST}/bin/apply_extra
+      - install -Dm755 minion.sh                  ${FLATPAK_DEST}/bin/minion.sh
+      - install -Dm644 ${FLATPAK_ID}.png          ${FLATPAK_DEST}/share/icons/hicolor/256x256/apps/${FLATPAK_ID}.png
+      - install -Dm644 ${FLATPAK_ID}.desktop      ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm644 ${FLATPAK_ID}.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
+    sources:
+      - type: extra-data
```

Comment:
Have you asked upstream if we could redistribute it using our CDN? This way users get delta upgrades instead of redownloading this file from scratch every time.

Reply:
There shouldn't be any delta upgrades at all as this is the final version of Minion 3, hasn't been for more than a year. When Minion 4 comes out I do think that it'll need essentially a new builder file altogether.

Reply:
I've also asked upstream for their github handles to add them when this get gets published. Usually I get a response within a couple of days but can't give an ETA. 

When they respond whether they want to have this on the flathub CDN I can easily add it is a dependency to be hosted on the flathub CDN. The download in `apply-extra` is only 7.7 MiB.

Reply:
> There shouldn't be any delta upgrades at all as this is the final version of Minion 3, hasn't been for more than a year. When Minion 4 comes out I do think that it'll need essentially a new builder file altogether.

Is the plan to update this ID to Minion 4 once it's released, or is this going to be a new submission?